README.md

```diff
@@ -118,9 +118,23 @@ You can access snapshot builds from the sonatype snapshot repository by adding t
 ```
 
 ## Build
+
+### Requirements
+
+* [Node.Js](https://nodejs.org/en/download/)
+* Java JDK >= 1.8
+* [Lombok Java](https://projectlombok.org) Library (debian pacakge `liblombok-java`)
+* Maven, with the plugins:
+    * [JavaDoc](https://maven.apache.org/plugins/maven-javadoc-plugin/) (debian package `libmaven-javadoc-plugin-java`)
+
+### Environment Variables
+
+* `PATH` must include Node.js's `npm` binary
+* `JAVA_HOME` is set to location of Java 1.8 (in Ubuntu this is
```

Comment:
Imho this is self-explanatory as jdk > 1.8 required

Reply:
Probably self-explanatory if you work on Java a lot of the time. IMHO, build docs should take the premise of "if you're new to 90% of what's used in this project, this will get you going." Docs like these take up only a little space, but can save someone a lot of work.